.github/workflows/mypy.yml

```diff
@@ -24,3 +24,5 @@ jobs:
       run: mypy --show-column-numbers --hide-error-context .
     - name: black
       run: black --check .
+    - name: Unit Tests
```

Comment:
Wondering if it makes sense to make the tests a separate workflow? As it stands, i think we wouldn't run the tests if mypy or black failed? Don't feel strongly about this, so if anyone else wants to weigh in please do 🙂

Reply:
that sounds good. but wouldn't a separate workflow run in parallel with this one? we'd have to put in some explicit dependency, which I am sure is possible, but I've never done

Reply:
ignore above comment. yes, this should be parallel so it always runs.

Reply:
I updated it so the tests are in parallel, works fine on my fork.